consensus/src/rpc/nimbus_rpc.rs

```diff
@@ -28,15 +43,7 @@ impl ConsensusRpc for NimbusRpc {
             self.rpc, root_hex
         );
 
-        let client = reqwest::Client::new();
-        let res = client
-            .get(req)
-            .send()
-            .await
-            .map_err(|e| RpcError::new("bootstrap", e))?
-            .json::<BootstrapResponse>()
-            .await
-            .map_err(|e| RpcError::new("bootstrap", e))?;
+        let res: BootstrapResponse = get(&req).await?;
```

Comment:
Looks great. Can we just make sure that the error types here are all `RpcError` like they were originally. It makes it easier when something goes wrong to track from exactly which rpc call it came from.